### PR TITLE
[DO NOT MERGE] v0.24.1.dev0

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   name: featuretools
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/featuretools/featuretools-{{ version }}.tar.gz
-  sha256: 32a7f1ae84ababa757a46ccef1695f0843b0f224c5aae4a1ddfffb6e4c033351
+  sha256: fa0b444055f8bb4046d79823f4b0226b740cbc1ad1016b2ca80c35138168ec29
 
 build:
   entry_points:
```
Comment (gsheni) [inline recipe/meta.yaml:12]: We made changes to requirements.txt, and test-requirements.txt in Featuretools, and those should be reflect here.
For example, scipy min was upgraded, pandas min was upgraded

Comment (jeff-hernandez) [reply recipe/meta.yaml:12]: Ah okay, thanks

File: recipe/meta.yaml [hunk 2]
```diff
@@ -42,9 +42,9 @@ test:
     - featuretools
   requires:
     - boto3 >=1.17.46
-    - composeml >=0.2.0
+    - composeml >=0.3.0
     - fastparquet >=0.1.6
```
Comment (gsheni) [inline recipe/meta.yaml:46]:
```suggestion
    - fastparquet==0.5.0
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -42,9 +42,9 @@ test:
     - featuretools
   requires:
     - boto3 >=1.17.46
-    - composeml >=0.2.0
-    - fastparquet >=0.1.6
-    - moto >=1.3.13
+    - composeml >=0.3.0
+    - fastparquet==0.5.0
+    - moto >=1.3.15
     - pip >=0.20.2
```
Comment (jeff-hernandez) [inline recipe/meta.yaml:48]: Noticed this pip version is higher than `19.0.2`

Comment (jeff-hernandez) [reply recipe/meta.yaml:48]: Nvm, will update

File: recipe/meta.yaml [hunk 4]
```diff
@@ -42,10 +42,10 @@ test:
...
-    - fastparquet >=0.1.6
-    - moto >=1.3.13
-    - pip >=0.20.2
+    - composeml >=0.3.0
+    - fastparquet==0.5.0
+    - moto >=1.3.15
+    - pip >=19.0.2
     - pympler >=0.8
     - pytest >=5.2.*
     - python-graphviz >=0.8.4
```
Comment (gsheni) [inline recipe/meta.yaml:51]: Add pyarrow
```
pyarrow >=3.0.0
```

Comment (gsheni) [reply recipe/meta.yaml:51]: @jeff-hernandez @rwedge should we add wheel or no?
```
wheel>=0.33.1
```

Comment (gsheni) [reply recipe/meta.yaml:51]: actually, it's its not needed, let's not put it in.

**Review by gsheni — APPROVED** [review]

**Pull request closed without merging**